packages/react-components/src/components/AutoComplete/helpers.tsx

```diff
@@ -0,0 +1,40 @@
+import * as React from 'react';
+
+import { IPickerListItem } from '../Picker';
+
+import { AutoCompleteProps, IAutoCompleteListItem } from './types';
+
+export const areAllOptionsStrings = (
+  options: AutoCompleteProps['options']
+): options is string[] =>
+  options !== undefined && typeof options[0] === 'string'; // Assumption (backed by TS): if o[0] is a string, the rest are too.
+
+export const buildOptionsFromStrings = (options: string[]): IPickerListItem[] =>
+  options.map((option) => ({ name: option, key: option }));
+
+export const buildOptionsFromAutoCompleteListItems = (
+  options: IAutoCompleteListItem[]
+): IPickerListItem[] =>
+  options.map(({ customElement, name, ...rest }) => ({
+    name,
+    key: name,
```

Comment:
What if somebody passes options with duplicated names?

Reply:
what if someone passes options with duplicated keys to `Picker`? I don't think we check that anywhere

Reply:
Yes, but keys should be unique by design, while names don't need to be. I mentioned this because we previously encountered an issue with the Picker, where we used names as keys, which caused problems with items rendering.

Reply:
But taking into account your other reply I think we can leave it like this 👌